pkg/yurtctl/cmd/markautonomous/markautonomous.go

```diff
@@ -29,9 +29,9 @@ import (
 	"k8s.io/klog/v2"
 
 	"github.com/openyurtio/openyurt/pkg/projectinfo"
-	"github.com/openyurtio/openyurt/pkg/yurtctl/constants"
+	"github.com/openyurtio/openyurt/pkg/yurtadm/constants"
```

Comment:
For `constants`, how about retaining the `constants` pkg both in `yurtctl` and `yurtadm`?
We can extract constants what yurtadm and yurtctl need respectively in `pkg/yurtadm/constants` and `pkg/yurtctl/constants`.
And each cmd only needs to import the constants pkg under its folder.

Reply:
Many constants is common, may also need to put the part in `pkg/util`.

Reply:
You can have a try, I don't find so many common constants. If there're only several common constants used by both yurtctl and yurtadm, the duplication is bearable.

Reply:
We only use the `constants.AnnotationAutonomy` in yurtctl, please move it from `pkg/yurtadm/constants` to `pkg/yurtctl/constants` and import `pkg/yurtctl/constants` instead.